lib/Pipelines/ArithmeticPipelineRegistration.h

```diff
@@ -28,6 +28,12 @@ void heirSIMDVectorizerPipelineBuilder(OpPassManager &manager,
                                        bool disableLoopUnroll);
 
 struct MlirToRLWEPipelineOptions : public SimdVectorizerOptions {
+  PassOptions::Option<bool> skipArithmetization{
+      *this, "skip-arithmetization",
```

Comment:
Generally speaking it's better to have all flags be phrased in terms of enabling a feature, rather than have some flags enable and some flags disable. Could we change this to `enable-automated-arithmetization` with a default of true?